javascript/javascript1/week1/ez-name.js

```diff
@@ -0,0 +1,9 @@
+let firstWords = ["Easy","Awesome","Corporate","Financial","Oracle", "nets","Internet","Vikings","Dansk","Indisk" ];
+let secondWords = ["Hard","Triocon","Webmaster","Developer","Leo", "Novo","Nordics","Nyt","Norwegien","Tysk"];
+
+const randomNumber = Math.floor(Math.random() * 10);
+
+let startupName = firstWords[ randomNumber] +' '+ secondWords [randomNumber];
+let startupNameLength = startupName.length;
+
+console.log("The startup: " + startupName+" contains " +startupNameLength+" characters");
```

Comment:
you can directly replace startupNameLength with startupName.length (to avoid another variable decleation)